test/extensions/ERC721ABurnable.test.js

```diff
@@ -0,0 +1,133 @@
+const { expect } = require('chai');
+const { constants } = require('@openzeppelin/test-helpers');
+const { ZERO_ADDRESS } = constants;
+
+describe('ERC721ABurnable', function () {
+  beforeEach(async function () {
+    this.ERC721ABurnable = await ethers.getContractFactory('ERC721ABurnableMock');
+    this.token = await this.ERC721ABurnable.deploy('Azuki', 'AZUKI');
+    await this.token.deployed();
+  });
+    
+  beforeEach(async function () {
+    const [owner, addr1, addr2] = await ethers.getSigners();
+    this.owner = owner;
+    this.addr1 = addr1;
+    this.addr2 = addr2;
+    await this.token['safeMint(address,uint256)'](this.addr1.address, 10);
+    await this.token.connect(this.addr1).burn(5);
```

Comment:
can you make 5 here a constant called `burnedTokenId` and use that in the tests below to make it clear what you're testing for? I want to avoid magic numbers where we can